hashes/hamming_code.py

```diff
@@ -77,6 +77,10 @@ def emitter_converter(size_par, data):
 
     >>> emitter_converter(4, "101010111111")
     ['1', '1', '1', '1', '0', '1', '0', '0', '1', '0', '1', '1', '1', '1', '1', '1']
+    >>> emitter_converter(5, "101010111111")
```

Comment:
Have you tried running `python -m doctest -v hashes/hamming_code.py`

Reply:
I did, when I run it it shows this:

1 items had no tests:
    hamming_code
4 items passed all tests:
   1 tests in hamming_code.emitter_converter
   1 tests in hamming_code.receptor_converter
   1 tests in hamming_code.text_from_bits
   1 tests in hamming_code.text_to_bits
4 tests in 5 items.
4 passed and 0 failed.
Test passed.

But I see below that when you ran it there were tests failing in this file. Any guidance on what I might be doing wrong that tests are passing for me, but not when someone else is running it? Thank you for your help! This is my first attempt at an open source contribution so I truly appreciate your help navigating all of this :)